config/Betterloadingscreen/tips/en_US.txt

```diff
@@ -106,10 +107,11 @@ With the power of Duct Tape, I solved all of your maintenance problems!
 When you open a Twilight Portal, you'll get hit by lightning
 Sometimes the best combat strategy is "Run away!"
 You can power machines by putting batteries in their energy slots
+You can power machines by putting Redstone in their energy slots
```

Comment:
if anything we should remove that quirk at some point. I dont think we want to encourage it.